test/central_directory_test.rb

```diff
@@ -57,10 +57,13 @@ def test_write_to_stream
 
   def test_write64_to_stream
     ::Zip.write_zip64_support = true
-    entries = [::Zip::Entry.new('file.zip', 'file1-little', 'comment1', '', 200, 101, ::Zip::Entry::STORED, 200),
-               ::Zip::Entry.new('file.zip', 'file2-big', 'comment2', '', 18_000_000_000, 102, ::Zip::Entry::DEFLATED, 20_000_000_000),
-               ::Zip::Entry.new('file.zip', 'file3-alsobig', 'comment3', '', 15_000_000_000, 103, ::Zip::Entry::DEFLATED, 21_000_000_000),
-               ::Zip::Entry.new('file.zip', 'file4-little', 'comment4', '', 100, 104, ::Zip::Entry::DEFLATED, 121)]
+    entries = [
+      ::Zip::Entry.new('file.zip', 'file-tiny', 'comment', Zip::Entry::STORED),
+      ::Zip::Entry.new('file.zip', 'file-big', 'comment'),
+      ::Zip::Entry.new('file.zip', 'file-alsobig', 'comment'),
+      ::Zip::Entry.new('file.zip', 'file-little', 'comment')
```

Comment:
It looks like these test entries have lost their sizes. From the last line of the test, it seems that the local header offsets are enough to force zip64, but it seems like the test is perhaps a bit less realistic without the sizes?

Reply:
These revert back to how they were with the keyword argument API.